pkg/api/index.go

```diff
@@ -53,14 +53,14 @@ func (hs *HTTPServer) getProfileNode(c *models.ReqContext) *dtos.NavLink {
 	}
 
 	return &dtos.NavLink{
-		Text:         c.SignedInUser.NameOrFallback(),
-		SubTitle:     login,
-		Id:           "profile",
-		Img:          gravatarURL,
-		Url:          hs.Cfg.AppSubURL + "/profile",
-		HideFromMenu: true,
```

Comment:
yeah, now we've added the `Section` property to the struct, this `HideFromMenu` becomes redundant and a little confusing (since at the top level it never hid anything from the menu, it was used to control what appeared in `TopSection`/`BottomSection`).

Reply:
So when the feature toggle is the new "default" we need to revisit this and deprecate/remove `HideFromMenu`.

Reply:
yeah, we probably need to separate out `NavLink` and create `NavLinkChild` or something. for `NavLinkChild` it makes sense and behaves as expected. for a top level `NavLink` it was kinda being repurposed